src/common/models/Projection.ts

```diff
@@ -327,6 +333,22 @@ export class Projection {
     return getPercentChange(firstVal, lastVal);
   }
 
+  private getHospitalizationInfo(): HospitalizationInfo | null {
+    const latestDatapoint = this.smoothedHospitalizations[
+      this.smoothedHospitalizations.length - 1
+    ];
```

Comment:
I think if the hospitalization data is at all lagging (e.g. we have data for 2 days ago but not yesterday) then this will fail.  This is probably why counties aren't showing up right now.  County data often lags by ~3 days.  There's a `lastValue()` helper that's used throughout this file that I think should work.

Reply:
ahhhhh thanks!